api/client/stack/remove.go

```diff
@@ -63,6 +63,11 @@ func runRemove(dockerCli *client.DockerCli, opts removeOptions) error {
 		}
 	}
 
+	if len(services) == 0 && len(networks) == 0 {
```

Comment:
This is part of the `stack rm` command which also removes the networks, so if a network existed this command wouldn't be a no-op, hence no warning. I believe that's the idea.